docs/backup/performance/database-indexes.md

```
@@ -0,0 +1,144 @@
+
+**Purpose**: Optimizes the feed refresh scheduler query that finds active feeds due for refresh.
+
+**Query Pattern**:
+```sql
+SELECT * FROM feeds 
+WHERE is_active = TRUE AND next_refresh <= CURRENT_TIMESTAMP
+ORDER BY next_refresh ASC
+LIMIT ?
+```
+
+**Impact**: 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)